doc/Makefile

```diff
@@ -142,11 +151,11 @@ prunedocs: FORCE
 	fi
 
 
-clean: clean-source clean-build clean-build-dir FORCE
+clean: clean-source clean-build clean-build-dir clean-doxygen FORCE
 
-cleanall: clean-source clean-build clean-build-dir FORCE
+cleanall: clean-source clean-build clean-build-dir clean-doxygen FORCE
 
-clobber: clean-source clean-build clean-build-dir FORCE
+clobber: clean-source clean-build clean-build-dir clean-doxygen FORCE
```

Comment:
I don't think we should call `clobber-libchplcomp` here because that will remove the cmake configuration and built .o files in additional to the doxygen materials. I think this clobber should only impact the input/artifacts for the docs build.

Reply:
Isn't that what led us to cases like "this file has been deleted but it still ends up in the build directory and these sorts of programs complain when pages are unreachable"?  Don't we want to avoid that?